src/lib/icons/IconCheckCircleFill.svelte

```diff
@@ -0,0 +1,18 @@
+<!-- source: DFINITY foundation -->
+<script lang="ts">
+  export let size = `24px`;
```

Comment:
is this intended? shouldn't it be `DEFAULT_ICON_SIZE`?

Reply:
Good catch! I blindly copied it from IconCheckCircle.svelte. It’s now updated.